src/components/spatial-query/index.js

```diff
@@ -0,0 +1,148 @@
+import PropTypes from 'prop-types';
+import React from "react";
+import cx from "classnames";
+
+class SpatialQuery extends React.Component {
+  constructor(props) {
+    super(props);
+
```

Comment:
Can we store these values with numbers instead of strings?

Reply:
It seems strange to write strings in there when we actually want to represent floating point numbers

Reply:
Or maybe you're using them as placeholders to indicate empty?  I'm not quite sure I understand.

Reply:
Yeah, these values are just the placeholders to indicate empty.

Reply:
I don't think we should use 0 or some numbers to represent the empty values.

Reply:
OK, I think that makes sense.  Maybe just add a comment indicating that's why we're doing it and write that the actual values that get stored are numbers.

Reply:
Ok